### DISCUSSION TITLE
Install with `ant-design-vue@next`, got version 2 instead of 3

### DISCUSSION THREAD

**Discussion by fahmiegerton** [vueComponent/ant-design-vue#5141] [Q&A]
Hi, I just realize something that quite bothering me that need to be asked. I install the version 3 with `ant-design-vue@next` as the docs said, but I notice when I check the version with yarn, it install the version 2 instead of 3. Is this like this or something I miss? Thanks
![image](https://user-images.githubusercontent.com/5303871/148798023-28958301-82f2-4df0-a06f-ae15c9f7b7b3.png)

**Comment by tangjinzhou** [comment]
ant-design-vue@3.0.0-0

**Reply by fahmiegerton** [reply]
I'm not sure what you mean, but according to npmjs, the `next` tag refer to version 2, 
![image](https://user-images.githubusercontent.com/5303871/149661679-140c074e-5a34-4da9-baaa-671de49c012d.png)

while to use the version 3, we need to use the `beta` tag (which is not in the docs)
![image](https://user-images.githubusercontent.com/5303871/149661657-6bf49753-cc26-4959-aa2e-a94d3bd03be8.png)

I think the docs should be point to use the `beta` tag (`yarn add ant-design-vue@beta`) to use version 3 instead of `next` (which currently in the docs and got installed the version 2). That's just my suggestion, or maybe the version 3 will be out of beta soon?
